janusgraph-core/src/main/java/org/janusgraph/graphdb/management/JanusGraphManager.java

```diff
@@ -89,6 +96,35 @@ public static JanusGraphManager getInstance(boolean forceCreate) {
         }
     }
 
+    public void configureGremlinExecutor(GremlinExecutor gremlinExecutor) {
+        this.gremlinExecutor = gremlinExecutor;
+        final ScheduledExecutorService bindExecutor = Executors.newScheduledThreadPool(1);
+        // Dynamically created graphs created with the ConfiguredGraphFactory are
+        // bound across all nodes in the cluster and in the face of server restarts
+        bindExecutor.scheduleWithFixedDelay(new GremlinExecutorGraphBinder(this.gremlinExecutor), 0, 20L, TimeUnit.SECONDS);
```

Comment:
It would be awkward to do this, as this is not tied to a specific graph and the only way to inject configuration settings into the JG server is by using graph configuration options.

We could make it a graph config option, but it would be awkward cause it would only be relevant upon the call to `configureGremlinExecutor` which would happen once at server start. We could make it only a config option read off of the `ConfigurationManagementGraph`'s configuration settings, but I worry that will be confusing as well.

Reply:
Thanks. I'm ok with leaving it as a constant. I don't think there is an immediate need to configure it.